Inspicio/Models/Review.cs

```diff
@@ -14,9 +14,11 @@ public class Review
 
         public int ImageId { get; set; }
 
-        public Boolean Liked { get; set; }
+        public Boolean Approved { get; set; }
```

Comment:
Using `Boolean` looks odd in a C# codebase, the keyword `bool` is normally what would be used - `Boolean` makes the code look like Java.

[[+_____]](https://gist.github.com/WilkaH/29df175966a76687f846)